packages/flutter/lib/src/services/hardware_keyboard.dart

```diff
@@ -457,6 +457,53 @@ class HardwareKeyboard {
   /// of the event.
   Set<KeyboardLockMode> get lockModesEnabled => _lockModes;
 
+  /// Returns true if the given [LogicalKeyboardKey] is pressed, according to
+  /// the [HardwareKeyboard].
+  bool isLogicalKeyPressed(LogicalKeyboardKey key) => logicalKeysPressed.contains(key);
```

Comment:
Is it faster to just write `_pressedKeys.containsKey` and `_pressedKeys.containsValue`?

Reply:
Maybe, but it's easier to migrate if the names don't change.

Reply:
(or are similar, I guess)

Reply:
No, I'm just talking about the internal implementation.

Reply:
Oh, duh. :-)

Yeah, OK, I'll do that.